website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Vadim Chelyshov
+title: Towards better releases
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+As many of you might have noticed, the previous `0.11.0` release didn't go smoothly due to a number of issues that came to light only after the release was published.
+Some of them were quite critical and the only option to continue to work was to downgrade Metals until `0.11.1` arrived.
+
+We apologize for that!
+
+However, this post is not only about saying sorry.
+The reasons why it's happened weren't a complete surprise for us.
+In order to avoid such situations in the future we have taken upon us to take steps that will allow us to detect issues early.
+Some of these issues were quite obvious and we were aware of them for a while.
+
+There were two main improvements that were implemented since the last release:
+- Backpublish Scala versions support
+- Pre-release version for VSCode extension
+
+## Backpublish Scala versions support
+
+We haven't mentioned this in the release notes for `0.11.0`, but from that version we have changed the way Metals detect whether a specific version is supported and thanks to that we are now able to backpublish new Scala versions support.
+The support of Scala `3.1.1` Scala was added to Metals `0.11.1` using this new mechanic.
+It allows us to provide new Scala versions support much faster than before and also support Scala3-NIGTHTLY versions.
+
+The additional benefit from this approach is that it eliminates time pressure from future Metals releases.
+If you look at release notes for previous versions, almost every one brings at least one new compiler version support.
+Having this limitation and the need to provide the new release as soon as possible it always was increasing the chances to affect the quality of release. So, now with this new feature we are able to take our time for final fixes and do release with more confidence.
```

Comment:
I would move this part to the beginning, this will allow creating previously/currently structure of the paragraph. Also I would mention a bit more about why Metals need compiler-specific things. Sth like that:

In order to provide (features?/completions? - I can't recall the proper word) Metals use Scala version-specific compiler API, that's why previously we had to release Metals for each new Scala version.

Then there will be a nice flow: how it was working, why it was a problem and what is the solution.

Reply:
To cover these both changes I come with the following structure:
```
- Beginning - introduce a problem and quickly mention that next parts are covering each solution that was implemented
- More detailed text per every feature
```

If I move this part to then last two paragraphs won't make sense.